rfc/composite_table_availability.pseudocode

```diff
@@ -0,0 +1,116 @@
+
+PhysicalTableSchema:
+	getFieldName(Column c)
```

Comment:
The `FieldName` mentioned here refers to things with a `asName()` ? like `Columns`?